### DISCUSSION TITLE
[Feature Request] Add support for Google Analytics v4

### DISCUSSION THREAD

**Discussion by hexsix** [Lednerb/bilberry-hugo-theme#387] [Ideas]
Now Google Analytics v4 is created by default.

[https://gohugo.io/templates/internal/#google-analytics](https://gohugo.io/templates/internal/#google-analytics)

**Comment by igor-baiborodine** [comment]
@hexsix, moved to Discussions/Idea section.

**Comment by igor-baiborodine** [comment]
Apparently, the min Hugo version should be bumped to 0.82.0 : https://github.com/adityatelange/hugo-PaperMod/issues/332

The PaperMod theme has support for GA v4: https://github.com/adityatelange/hugo-PaperMod/blob/master/layouts/partials/head.html

Proposed solution: add `{{- template "_internal/google_analytics.html" . }}` to the `baseof.html` layout. 
To see, if the `{{ template "_internal/google_analytics_async.html" . }}` line should be removed or kept for the compatibilty.

**Comment by igor-baiborodine** [comment]
@hexsix, implemented with PR https://github.com/Lednerb/bilberry-hugo-theme/pull/388

**Reply by hexsix** [reply]
Thank you!
